pkg/endpoints/filters/authentication.go

```diff
@@ -50,15 +50,15 @@ func WithAuthentication(handler http.Handler, auth authenticator.Request, failed
 		defer recordAuthMetrics(resp, ok, err, apiAuds, authenticationStart)
 		if err != nil || !ok {
 			if err != nil {
-				klog.Errorf("Unable to authenticate the request due to an error: %v", err)
+				klog.V(5).Infof("Unable to authenticate the request due to an error: %v", err)
```

Comment:
I think that auth err log is necessary, do we need to adjust log level?

Reply:
There are too many logs here when gateway receives invalid token.
But after thinking deeply, I will revert this file now. It seems that we should reduce the number of requests with invaid token firstly